components/HackKnightChallenge/KnightChallenge.module.scss

```diff
@@ -0,0 +1,153 @@
+.container {
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    row-gap: 2rem;
+}
+
+.box {
+    background-color: #a61e00;
```

Comment:
the contrast is a bit harsh imo, i asked design to give us a new background color to make the challenge more readable